### PR TITLE
Adding cores parameter to use

### PR DESCRIPTION
**Pull request by senthil10**
Intent for this PR is explained in this [issues](https://github.com/SciLifeLab/TACA/issues/76). 
Thanks @guillermo-carrasco , for suggesting a very easy fix :)

### CODE REVIEW

File: taca/storage/cli.py [hunk 1]
```diff
@@ -16,13 +16,14 @@ def storage(ctx, days, run):
 @storage.command()
 @click.option('--backend', type=click.Choice(['swestore']), required=True,
               help='Long term storage backend')
+@click.option('-c','--cores', type=click.INT, help='Limit the number of cores used simultaneously')
```
Comment (guillermo-carrasco) [inline taca/storage/cli.py]: I would prefer to clarify that this is about the number of runs you are going to archive simultaneously. The user doesn't have to care about cores. The implementation is perfect, I would just change:

```
@click.option('-m','--max-runs', type=click.INT, help='Limit the number of runs to be archived simultaneously')
```

vad tror du?

Comment (ewels) [inline taca/storage/cli.py]: CF :heart: `--max_runs`
http://clusterflow.io/cl_reference/#_8

**Comment by senthil10** [comment]
@guillermo-carrasco : makes sense, for a minute thought to be a computer guy using computer terms :stuck_out_tongue: Made a commit with changes :+1:

**Comment by guillermo-carrasco** [comment]
bra jobbat!

**Pull request merged**